src/git-ship

```diff
@@ -38,6 +43,9 @@ function preconditions {
   ensure_is_feature_branch "$target_branch_name" "Only feature branches can be shipped."
   ensure_knows_parent_branches "$target_branch_name"
   ensure_has_no_unshipped_ancestors "$target_branch_name"
+  if [ "$HAS_REMOTE" = true -a "$fetched" = false ]; then
```

Comment:
We use two different styles for boolean and in this method now:

```bash
if [ "$#" -gt 0 ] && [[ ! "$1" =~ ^- ]]; then
```

as well as

```bash
if [ "$HAS_REMOTE" = true -a "$fetched" = false ]; then
```

Let's agree on one style and use it consistently.